modules/apps/object/object-web/src/main/resources/META-INF/resources/js/utils/exportObjectFolder.ts

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import {createResourceURL} from 'frontend-js-web';
```

Comment:
can you please squash this into the respective commit it belongs?

Reply:
Done 😄

Reply:
Hi @rebsilva, I just realized that my rebase wasn't quite successful.

I will sent another push force to move this change to respective commit.

Reply:
Hi @rebsilva, I just realized that my rebase wasn't quite successful.